src/Forms/Shared/Samples/Layers/QueryCQLFilters/QueryCQLFilters.xaml.cs

```diff
@@ -0,0 +1,217 @@
+
+        // Note that the service defines the collection id which can be accessed via OgcFeatureCollectionInfo.CollectionId.
+        private const string CollectionId = "TransportationGroundCrv";
+
+        public QueryCQLFilters()
+        {
+            InitializeComponent();
+            _ = Initialize();
+        }
+
+        private async Task Initialize()
+        {
+            // Populate the UI.
+            WhereClausePicker.ItemsSource = DefaultWhereClause;
+            MaxFeaturesBox.Text = "3000";
+            StartDatePicker.Date = new DateTime(2011, 6, 13);
+            EndDatePicker.Date = new DateTime(2012, 1, 7);
+
+            // Create the map with topographic basemap.
+            MyMapView.Map = new Map(BasemapStyle.ArcGISTopographic);
+
+            try
+            {
+                LoadingProgressBar.IsVisible = true;
+
+                // Create the feature table from URI and collection id.
+                _featureTable = new OgcFeatureCollectionTable(new Uri(ServiceUrl), CollectionId);
+
+                // Set the feature request mode to manual (only manual is currently supported).
+                // In this mode, you must manually populate the table - panning and zooming won't request features automatically.
+                _featureTable.FeatureRequestMode = FeatureRequestMode.ManualCache;
+
+                // Load the table.
+                await _featureTable.LoadAsync();
+
+                // Populate the OGC feature collection table.
+                QueryParameters queryParamaters = new QueryParameters();
+                queryParamaters.MaxFeatures = 3000;
+                await _featureTable.PopulateFromServiceAsync(queryParamaters, false, null);
+
+                // Create a feature layer from the OGC feature collection
+                // table to visualize the OAFeat features.
+                FeatureLayer ogcFeatureLayer = new FeatureLayer(_featureTable);
+
+                // Choose a renderer for the layer based on the table.
+                ogcFeatureLayer.Renderer = GetRendererForTable(_featureTable) ?? ogcFeatureLayer.Renderer;
+
+                // Add the layer to the map.
+                MyMapView.Map.OperationalLayers.Add(ogcFeatureLayer);
+
+                // Zoom to the extent of the feature layer/table.
+                Envelope tableExtent = _featureTable.Extent;
+                if (tableExtent != null && !tableExtent.IsEmpty)
+                {
+                    await MyMapView.SetViewpointGeometryAsync(tableExtent, 20);
+                }
+            }
+            catch (Exception ex)
+            {
+                await Application.Current.MainPage.DisplayAlert("Error", ex.Message, "OK");
+            }
+            finally
+            {
+                // Update the UI.
+                LoadingProgressBar.IsVisible = false;
+            }
+        }
+
+        private Renderer GetRendererForTable(FeatureTable table)
```

Comment:
Is this necessary? It seems like the result will always be a polyline.

Reply:
This was there in case the user swapped out default data for their own. Considering that they would have to change url, collection id, and where clauses anyway, I think its good to simplify this and let the user change the renderer if necessary.